kymatio/scattering1d/filter_bank.py

```diff
@@ -511,9 +511,10 @@ def scattering_filter_factory(J_support, J_scattering, Q, T, r_psi=math.sqrt(0.5
     J_scattering : int
         parameter for the scattering transform (2**J_scattering
         corresponds to maximal temporal support of any filter)
-    Q : int
-        number of wavelets per octave at the first order. For audio signals,
-        a value Q >= 12 is recommended in order to separate partials.
+    Q : tuple
+        number of wavelets per octave at the first and second order 
+        Q = (Q1, Q2). For audio signals, a value Q1 >= 12 is recommended 
+        in order to separate partials. Q2 is optional and defaults to 1.
```

Comment:
Consider removing:

> Q2 is optional and defaults to 1.

We don't need to give the default value in the `scattering_filter_factory`, only in the constructor